internal/cli/clients/client.go

```diff
@@ -580,12 +573,14 @@ func (c *EpinioClient) Info() error {
 		return errors.Wrap(err, "failed to get kube version")
 	}
 
+	// TODO: Extend epinio API to get this information. Or remove this information.
+
 	giteaVersion := "unavailable"
 
-	version, resp, err := c.GiteaClient.Client.ServerVersion()
-	if err == nil && resp != nil && resp.StatusCode == 200 {
-		giteaVersion = version
-	}
+	// version, resp, err := c.GiteaClient.Client.ServerVersion()
+	// if err == nil && resp != nil && resp.StatusCode == 200 {
+	// 	giteaVersion = version
+	// }
```

Comment:
delete the comments? The TODO above should be enough to fix this.

Reply:
Right. I left quite a bit in regarding the gitea version for info ...

Reply:
Done, 2ceb59f79c32d91a623818ebb927e5e855a6f709